src/platform/datapath_winkernel.c

```diff
@@ -765,6 +766,25 @@ CxPlatDataPathQuerySockoptSupport(
 
     } while (FALSE);
 
+    do {
```

Comment:
IMO, a do/while loop here is overkill. You could have just had the if statement, and only added the feature flag if the status was success and build not equal to 20348.